MOVIShield.cpp

```diff
@@ -45,11 +45,13 @@
 
 MOVI::MOVI()
 {
+    usehardwareserial=false;
     construct(ARDUINO_RX_PIN, ARDUINO_TX_PIN, false);
 }
 
-MOVI::MOVI(bool debugonoff)
+MOVI::MOVI(uint8_t debugonoff)
```

Comment:
Based on some quick research, it looks like implicit conversion from a pointer to bool is allowed and actually intended for uses such as:

if (p) {
...
}

Where the expression p returns false if p is NULL and true otherwise.

I imagine the compiler will choose the HardwareSerial * constructor for any pointer that is a HardwareSerial instance or derivative and the bool constructor for any other pointer but I haven't tested it.  I can try it if you'd rather keep bool instead of protecting from construction with arbitrary pointers using uint8_t.  Let me know.